### ISSUE TITLE
Red loop and error message "Pump data is -11,846,985 minutes old"

### ISSUE THREAD

**Issue by Oursweetpea** [LoopKit/Loop#1225]
We received a red loop about 40 minutes ago and cannot figure out what is wrong. Rileylink is connected, cgm is still working, pump is working. When you tap on the red loop you get the message that is attached below. 
Info:
CGM: Dexcom G6
Pump: Omnipod
Loop version: 1.10.0 omnipod testing branch
Phone ios: 13.2

[Loop issue report.pdf](https://github.com/LoopKit/Loop/files/4014661/Loop.issue.report.pdf)
![Screenshot 2020-01-01 at 6 18 24 PM](https://user-images.githubusercontent.com/57298157/71647624-b7ae7800-2cc7-11ea-9384-8da020f1a635.png)

**Comment by Oursweetpea** [comment]
I figured out how to fix it. I closed loop and dexcom and restarted the phone. I powered off rileylink and waited a little bit and then powered it back on. I got a green loop once rileylink reconnected. I just am not sure exactly went wrong with loop to give that weird message.

**Comment by rickggaribay** [comment]
This is a persistent problem with our rig and probably the biggest complaint from my daughter who has been looping since May 2019.

We see it several times a day. The steps you took work sometimes but not always. I’m truly interested if anyone else sees this often (docs address some of this) as it would be useful to determine if the is hardware or software or both.

Rick G. Garibay
@rickggaribay
http://rickgaribay.net
________________________________

**Comment by live4sw** [comment]
I have RL connectivity issues periodically and need to turn RL off and on again. Issue #1166 appears similar. The response from @dm61 seemed about as clear an explanation as anyone had. I am on the most recent dev and iOS 13.3 and it happens to me every so often but not daily, more like every week or two. My next step would probably be to switch to a new RL.

**Comment by danno-loop** [comment]
ditto to what @live4sw states....this is frustrating and still happens periodically to me as well without a known fix to my knowledge. Seems to affect both MM and Omnipod pumps from what I've seen across a few different, similar threads. Hoping this will get enough attention for a fix soon.

**Comment by rickggaribay** [comment]
Thanks. We actually have a backup RL new in box. Might give that a try. Thank you for the reply and the suggestion.

On Jan 1, 2020, at 5:37 PM, live4sw <notifications@github.com<mailto:notifications@github.com>> wrote:


I have RL connectivity issues periodically and need to turn RL off and on again. Issue #1166<https://github.com/LoopKit/Loop/issues/1166> appears similar. The response from @dm61<https://github.com/dm61> seemed about as clear an explanation as anyone had. I am on the most recent dev and iOS 13.3 and it happens to me every so often but not daily, more like every week or two. My next step would probably be to switch to a new RL.

**Comment by live4sw** [comment]
As another note, I've noticed my RL has randomly gotten hot every so often in the past month or so. This has coincided with the winter, and I've heard static electricity anecdotally mentioned as a potential culprit for RL issues. It seems (no data to support) that RL failure reports have increased as the weather has gotten colder, and there are many Loopers who have started in the past 6 months (myself included after 3 years on OA) who have never experienced a winter with an RL. So just throwing it out there.

I did replace my RL this morning and it has been working well so far.
